### PR TITLE
Change group subject to tags #230

### LINKED ISSUE #230: Change attribute subject to tags
#### Idea:
When creating a group, instead of defining the subject, the user should be able to add tags in form. See https://wikiki.github.io/form/tagsinput/ for the UI, and change db accordingly.
#### Reason:
There might be more than one subject of the event, and using tags is more efficient for the eyes to look through the groups. What's more, they should be easier to filter by.

### PR DESCRIPTION
**Pull request by triszt4n**
Closes #230 

#### What's new:
* Changed subject input to bulma-tagsinput
* Shows tags on group show page

#### Todo:
* Change migration
* Change name subject everywhere to tags.

### CODE REVIEW

**Comment by triszt4n** [comment]
This is in visualization stage, have a look at it, and make some comments. Thanks!
#### Ideas to consider:
* Show these tags on the index page as well
* Configure the tags: maximum tag number, maximum character number

File: views/group/new.pug [hunk 1]
```diff
@@ -6,13 +6,14 @@ block styles
...
+  form(method="POST", action="/groups", id="group-form")
     .field
       label.label Csoport neve
         input.input(type="text", name="name", placeholder="Analízis ZH1 készülés" required)
     .field
-      label.label Tárgy
-        input.input(type="text", name="subject", placeholder="Analízis 1")
+      label.label Címkék
+      .control
+        input.input(type="tags", name="subject", id="tags", placeholder="Adj hozzá címkéket...", value="gyakorlás")
```
Comment (OmTheTurtle) [inline views/group/new.pug:16]: I wouldn't put any default value here.

**Comment by OmTheTurtle** [comment]
Checked out locally, looks great! One small thing is, I'd increase the font size of the tags on `groups/:id` page, the current one is a bit hard to read.

File: views/group/new.pug (hunk 1 above)
Comment (triszt4n) [reply views/group/new.pug:16]: Oops, sorry, forgot about that.

**Comment by triszt4n** [comment]
PR #290 will solve this issue from now on.

**Pull request closed without merging**